teachopencadd/talktorials/T014_binding_site_detection/talktorial.ipynb

```diff
@@ -0,0 +1,2116 @@
+{
```

Comment:
<ul><li> "with the manually assigned KLIFS binding pocket" - I would not call this "manually" because this is programmatically inferred, maybe use "with the KLIFS binding pocket definition"?</li><li>"KLIFS thereby enables us to do systematic comparisons or analyses of the structures, chemical features, bound ligands and interactions to such of all available structures" &gt; "to such of"? "based on" maybe?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>First, included. Second rephrased.</p>